src/components/useHead.jsx

```
@@ -7,26 +7,26 @@ import useUpdateFavicon from '@/hooks/useUpdateFavicon'
 import useUpdateDocumentTitle from '@/modules/views/useUpdateDocumentTitle'
 import {
   buildFileOrFolderByIdQuery,
-  buildSharedDriveFolderQuery
+  buildSharedDriveFolderQuery,
```

ESLint

[error] Delete `,`

(prettier/prettier)